src/components/posts/PostsPage.js

```diff
@@ -36,28 +36,22 @@ const PostsPage = () => {
   };
 
   const signOut = (event) => {
-    event.preventDefault();
-    localStorage.setItem(process.env.REACT_APP_USER_TOKEN, '');
-    navigate('../', { replace: true });
-  };
+    event.preventDefault()
+    localStorage.setItem(process.env.REACT_APP_USER_TOKEN, '')
+    navigate("../", { replace: true });
+  }
 
-  return (
+  return(
     <>
-      <Header companyName={`Cohort Manager 2.0`} />
-      <section className='posts-section'>
-        <button id='user-signout-button' onClick={signOut}>
-          sign out
-        </button>
-        <p>Status: {postResponse.status}</p>
-        <PostForm handleSubmit={createPost} handleChange={handleChange} />
-        <ul className='posts-list'>
-          {posts.map((post, index) => (
-            <li key={index} className='post-item'>
-              {post.content}
-            </li>
-          ))}
-        </ul>
-      </section>
+    <Header companyName={`Cohort Manager 2.0`} />
+    <section className="posts-section">
+      <button id="user-signout-button" onClick={signOut}>sign out</button>
+      <p>Status: {postResponse.status}</p>
+      <PostForm handleSubmit={createPost} handleChange={handleChange}/>
+      <ul className="posts-list">
+        {posts.map((post, index) => <li key={index} className="post-item">{`${post.content} ${post.user.email}`}</li>)}
```

Comment:
Can we display user first name and last name rather than email? 

Also - try to avoid making formatting/code layout changes along with functional changes - it's hard to see what the actual functional change was here. It looks like this is the only actual change here - if so, it's worth being more explicit in the PR text so reviewers know what they are looking for.

Reply:
That's cool, but I still think my point about formatting stands - I tend to commit code formatting changes separatley from functional changes to make it easier for others to quuickly note what has changed.